packages/dx-react-grid/docs/reference/table-edit-row.md

```diff
@@ -52,7 +52,7 @@ tableBodyRows | Getter | Array&lt;[TableRow](table-view.md#table-row)&gt; | Rows
 editingRows | Getter | Array&lt;number &#124; string&gt; | IDs of the rows being edited
 addedRows | Getter | Array&lt;Object&gt; | The created rows
 changedRows | Getter | { [key: string]: Object } | Uncommitted changed rows
-getCellData | Getter | (row: [Row](grid.md#row), columnName: string) => any | The function used to get a cell's data
+getCellValue | Getter | (row: [Row](grid.md#row), columnName: string) => any | The function used to get a cell's data
```

Comment:
Should we update description?
'The function used to get a cell's data' => The function used to get a cell's value.